libs/wingc/src/type_check.rs

```diff
@@ -2241,6 +2273,16 @@ impl<'a> TypeChecker<'a> {
 						let new_class = self.hydrate_class_type_arguments(env, WINGSDK_MUT_MAP, vec![t]);
 						self.get_property_from_class(new_class.as_class().unwrap(), property)
 					}
+					Type::MutJson => {
+						todo!(); // TODO: Need to add SDK support for MutJson
+						 // let new_class = self.hydrate_class_type_arguments(env, WINGSDK_MUT_JSON, vec![self.types.json()]);
+						 // self.get_property_from_class(new_class.as_class().unwrap(),	property)
+					}
+					Type::Json => {
+						todo!(); // TODO: Need to add SDK support for Json
+						 // let new_class = self.hydrate_class_type_arguments(env, WINGSDK_JSON, vec![self.types.json()]);
+						 // self.get_property_from_class(new_class.as_class().unwrap(),	property)
+					}
```

Comment:
I don't think our Json spec is related to generics in any way. So I don't understand why we need to "hydrate" here.

Reply:
@yoav-steinberg - it's for accessing properties.
We want to support `.get(propertyName)` (and `.set` too) - to do that we will use the "baby macros" mechanism.
Supporting dot notation will only come later.

Reply:
So actually if we just return something like
```rust
Type::MutJson => {
		VariableInfo {
              _type: self.types.mut_json(),
              flight: env.flight,
              reassignable: true 
           }
}
Type::Json => {
     VariableInfo { 
           _type: self.types.json(),
           flight: env.flight,
           reassignable: false 
	}
}
```

Then we have dot notation. Since the jsified Json uses JS objects rather than maps. But Im not sure about how to determine the proper `flight`

Reply:
Actually I think @staycoolcall911 is right here. `hydrate_class_type_arguments` isn't only for "baby generics" it's also for returning some std class from the sdk for a given "native" wing type. So your original commented code seems more-or-less correct. We just need to pass an empty `type_params` vector because there are not type params here (`Json` isn't generic).
If this is still unclear lets try to have a session about this..